### PR TITLE
RM-7518: WxePostbackOutOfSequenceException is thrown when doubleclicking the pagination buttons of a BocList

### PR DESCRIPTION
**Pull request by iam-mholle**
https://re-motion.atlassian.net/browse/RM-7518

### CODE REVIEW

**Review by Stefan-Ilic — APPROVED** [review]
I can confirm that the fix works for the repro provided. That's where my usefulness ends here, though

File: Remotion/Web/Core/Res/HTML/SmartPage.js [hunk 1]
```diff
@@ -1178,6 +1178,8 @@ function SmartPage_Context(
       _submitState = submitState;
     else if (_submitState.IsAutoPostback && !isAutoPostback)
       _submitState.NextSubmitState = submitState;
+    else
+      _submitState.NextSubmitState = { CancelSubmit: true };
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/HTML/SmartPage.js:1182]: I'm thinking, for consistency reasons, we should add CancelSubmit=false to line 1169 and Submitter=null to line 1182. This should make it more explicit that we have considered the relevant attributes and decided to make them null/false.

Please do, re-check the double postback catch behavior and the BocList bugfix, then we should be done.

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

File: Remotion/Web/Core/Res/HTML/SmartPage.js (hunk 1 above)
Comment (iam-mholle) [reply Remotion/Web/Core/Res/HTML/SmartPage.js]: Tested it, works as intended.

**Review by MichaelKetting — APPROVED** [review]

**Comment by iam-mholle** [comment]
Added to master with cce74420b42baba8598504163f93281e53637e90.

**Pull request closed without merging**